src/app/Pages/TransactionHistory.tsx

```diff
@@ -17,11 +21,47 @@ export default function TransactionHistory(): JSX.Element {
   }
 
   return (
-    <div>
+    <Container>
       <TransactionOverview
         transactions={transactions}
         deleteTransaction={handleDeleteTransaction}
       />
-    </div>
+      <Link to="/">
+        <StyledButton>
+          <Icon>
+            <ArrowIconRight />
+          </Icon>
+          <ButtonText>Startseite</ButtonText>
+        </StyledButton>
+      </Link>
+    </Container>
   )
 }
+
+const Container = styled.div`
+  display: grid;
+  grid-template-rows: auto 60px;
+`
+
+const StyledButton = styled(Button)`
+  font-weight: bold;
+  font-size: 1rem;
+  background-color: steelblue;
+  border: none;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```

Comment:
Why do you use five columns, but only use the last one for the ButtonText?
Maybe “1fr 20%” would be easier?

Reply:
That´s a good idea, thank you Paul